src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
+import { CommentList } from '../CommentList/CommentList';
+import { Post } from '../types/Post';
+import { UserInfo } from '../UserInfo/UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      {comments?.length
+        ? (<CommentList comments={comments} />)
+        : (<b data-cy="NoCommentsMessage">No comments yet</b>)}
```

Comment:
```suggestion
      {comments?.length ? (
        <CommentList comments={comments} />
      ) : (
        <b data-cy="NoCommentsMessage">No comments yet</b>
      )}
```
looks better if format ternary in this way :arrow_up:  :)

Reply:
Not resolved. Fix formatting